src/directory-sync/interfaces/directory.interface.ts

```diff
@@ -1,3 +1,24 @@
+export type DirectoryType =
```

Comment:
Now that `DirectoryType` exists in this file I'm going to come back later in https://linear.app/workos/issue/USRLD-632/change-directorytype-to-directorytype-instead-of-string to update line 30 to point towards `DirectoryType` instead of `string` as well. 

I just want to make sure it's in a separate OR so it doesn't get lost and we include it in the changelog.